pkgs/top-level/all-packages.nix

```
@@ -1724,6 +1724,8 @@ with pkgs;
 
   deltachat-cursed = callPackage ../applications/networking/instant-messengers/deltachat-cursed { };
 
+  DelayArchitect = callPackage ../applications/audio/DelayArchitect { };
```

```suggestion
  delayarchitect = callPackage ../applications/audio/delayarchitect { };
```
I forgot to tell you but this also needs to be lowercase. Not sure if it's a good idea to have the result named `DelayArchitect`, with uppercase letters, either.